doc/src/modules/solvers/ode.rst

```diff
@@ -152,7 +154,7 @@ almost_linear
 
 linear_coefficients
 ^^^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::ode_linear_coefficients
+.. autofunction:: sympy.solvers.ode.single::LinearCoefficients
```

Comment:
Should this be `autoclass`?

Reply:
yes, this should be autoclass. will rectify it